scripts/setup-termux.sh

```diff
@@ -45,6 +48,13 @@ PACKAGES+=" valac"
 PACKAGES+=" xmlto"                      # Needed by git's manpage generation
 PACKAGES+=" zip"
 
-apt update
-apt dist-upgrade -y
-apt install -y $PACKAGES
+if [ "$TERMUX_APP_PACKAGE_MANAGER" = "apt" ]; then
+	apt update
+	apt dist-upgrade -y
+	apt install -y $PACKAGES
+elif [ "$TERMUX_APP_PACKAGE_MANAGER" = "pacman" ]; then
+	pacman -Sy $PACKAGES --needed --noconfirm
```

Comment:
Does it really not need `pacman -Syu`?

Reply:
Yes, I think it would be better to have `pacman -Syu`.